paper-input-char-counter.html

```diff
@@ -0,0 +1,96 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../paper-styles/paper-styles.html">
+
+<!--
+`<paper-input-char-counter>` is a character counter for use with `<paper-input-container>`.
+-->
+<dom-module id="paper-input-char-counter">
+
+  <style>
+
+    :host {
+      display: inline-block;
+
+      mixin(--paper-input-container-add-on);
+      mixin(--paper-input-container-add-on-font);
+
+      mixin(--paper-input-char-counter);
+    }
+
+  </style>
+
+  <template>
+
+    <span>[[charCounter]]</span>
+
+  </template>
+
+</dom-module>
+
+<script>
+
+(function() {
+
+  Polymer({
+
+    is: 'paper-input-char-counter',
+
+    enableCustomStyleProperties: true,
```

Comment:
I wish this would just be a default thing. My guess is that it will get deprecated but we'll still be removing vestigial `enableCustomStyleProperties` for years to come.